buildSrc/src/main/groovy/io.deephaven.java-classpath-conventions.gradle

```diff
@@ -31,13 +31,12 @@ configurations {
     fishDataGenerator.extendsFrom jdom
     fishNet.extendsFrom fishIo
     fishNumerics.extendsFrom fishBase
-    fishUtil.extendsFrom fishConfig
-    fishStats.extendsFrom fishUtil
+    fishUtil.extendsFrom fishConfig, fishIo
```

Comment:
I think this change should come out, and probably you should add fishIo to like 39 (for dhUtil).